source/_components/google_assistant.markdown

```diff
@@ -137,31 +137,31 @@ If you're not using Linux, you can use sites such as [this one](https://www.brow
 
 ### {% linkable_title Setup %}
 
-1. Create a new project in the [developer console](https://console.actions.google.com/).
-  a. Add/Import project
-  b. Click on `BUILD` on the `Smart home` card
+1. Create a new project in the [developer console](https://console.actions.google.com/).  
+  a. Add/Import project  
+  b. Click on `BUILD` on the `Smart home` card  
   c. Type in your home assistant url: `https://[YOUR HOME ASSISTANT URL:PORT]/api/google_assistant`, replace the `[YOUR HOME ASSISTANT URL:PORT]` with the domain / ip address and the port under which your Home Assistant is reachable.
   d. Click `Done`. Then click on `Overview`, which will lead you to the app details screen.
 2. You'll need to fill out most of the information on that page, but none of it really matters since you won't be addressing the App directly, only through the Smart Home functionality built into Google Assistant.
-3. The final item on that page `Account linking` is required for your app to interact with Home Assistant.
```

Comment:
This should read:
The final item on that page `Account linking` is required for your app to interact with Home Assistant. From the left menu under the `Advanced Options` select `Account Linking`. Select `No, I only want to allow account creation on my website`, Linking type if `OAuth`

Reply:
This mentions more explicitly where to setup `Account Linking`  and what `Linking type` to use.